worldguard-bukkit/src/main/java/com/sk89q/worldguard/bukkit/cause/Cause.java

```diff
@@ -134,8 +135,8 @@ public Object getRootCause() {
     @Nullable
     public Player getFirstPlayer() {
         for (Object object : causes) {
-            if (object instanceof Player) {
-                return (Player) object;
+            if (object instanceof Player p && !p.hasMetadata("NPC")) {
```

Comment:
any reason this one checks metadata directly instead of using the util method?

Reply:
No, i started the changes with individual checks for the metadata and used the Entities.isNPC() later.

This was an old relict from that. I changed some other metadata checks to that method too.